intelmq/bots/collectors/shodan/collector_stream.py

```diff
@@ -46,14 +47,28 @@ def init(self):
             self.api = shodan.Shodan(self.api_key,
                                      proxies=self.proxy)
         if isinstance(self.countries, str):
+            if self.countries and self.alert:
+                raise ValueError('both alert and country filters specified in Shodan collector.')
```

Comment:
```suggestion
                raise ValueError('Both alert and country filters specified. Please use only one selector.')
```
Add a hint how to solve the error. Remove bot name, it's already in the error message.